nautobot/docs/index.md

```diff
@@ -57,6 +57,10 @@ Nautobot is built on the [Django](https://djangoproject.com/) Python Web framewo
 | Task queuing       | Redis/Celery/django-rq          |
 | Live device access | NAPALM                          |
 
+The low level diagram displays how data travels through Nautobot's application stack.
+
+![Application stack diagram](/docs/media/nautobot_apllication_stack_low_level.png "Application stack diagram")
```

Comment:
Instead of:


<h3>Database</h3>
<h5>MySQL/PostgreSQL</h5>


which seems backwards, compared to the other boxes, this should probably be:

<h3>PostgreSQL / MySQL</h3>
<h5>Database</h5>